src/main/java/com/google/cloud/tools/managedcloudsdk/install/InstallerFactory.java

```diff
@@ -45,28 +47,39 @@
    *
    * @param installedSdkRoot path to the Cloud SDK directory
    * @param progressListener listener on installer script output
+   * @param environmentVariables Map of additional environment variables to be passed on to the
```

Comment:
delete "Map of ". It's redundant with what javaodc fills in automatically

Reply:
done.